src/entity/posts/utils.ts

```diff
@@ -352,6 +353,16 @@ interface CreateSharePostArgs {
   args: SharePostArgs;
 }
 
+export const determineSharedPostId = (post: Post | SharePost): string => {
+  if (post.type === PostType.Share) {
+    if (!post.title) {
```

Comment:
Could this be in one if statement, so we don't need to nest if in if?
```suggestion
  if (post.type === PostType.Share && !post.title) {
```

Reply:
Ah yeah when I wrote it, I felt there was another condition...
There wasn't 😅